lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -17,12 +17,29 @@ module FlashHelper
     }
     def display_flash_messages(key_matching = {})
       key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+      key_matching.default = :standard
 
-      flash.inject "" do |message, (key, value)|
-        message += content_tag :div, :data => { :alert => "" }, :class => "alert-box #{key_matching[key.to_sym] || :standard}" do
-          (value + link_to("&times;".html_safe, "#", :class => :close)).html_safe
+      capture do
+        flash.each do |key, value|
+          next if FoundationRailsHelper.configuration.ignored_flash_keys.include? key.to_sym
```

Comment:
I prefer this approach over adding a variable to the `display_flash_messages` method because it's more of a global setting and not something that will change in the app.  To be honest, I forgot about the `key_matching` variable.  Maybe we should move that to the config also.